airbyte-integrations/connectors/source-hubspot/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "source_hubspot"
 
 [tool.poetry.dependencies]
 python = "^3.10,<3.12"
-airbyte-cdk = "^6"
+airbyte-cdk = "6.45.1.post46.dev14423672753"
```

Comment:
TODO: Bump to stable before merge. Pending:

- https://github.com/airbytehq/airbyte-python-cdk/pull/349